R/createRunFolder.R

```diff
@@ -2,11 +2,11 @@
 #'
 #' Create a folder for the model to run in and copy required gams files there.
 #'
-#' @param path character vector, containing
+#' @param path character vector, containing the path(s) to the folder(s) to be created.
 #' @param config list with run configuration
-#' @param overwrite logical; Should exiting folders be overwritten?
-#' @param recursive logical; Should exiting folders be overwritten?
-#' @param showWarnings logical; Should exiting folders be overwritten?
+#' @param overwrite logical; Should existing folders be overwritten?
+#' @param recursive logical; Should elements of the path other than the last be created?
+#' @param showWarnings logical; Should a warning on not created paths be shown?
```

Comment:
Thanks look good. I'm still irritated by the semicolon that I once put there for some reason. As you change these lines now, maybe you can also replace those?
```suggestion
#' @param overwrite logical, Should existing folders be overwritten?
#' @param recursive logical, Should elements of the path other than the last be created?
#' @param showWarnings logical, Should a warning on not created paths be shown?
```